doc/ip/pinmux.md

```diff
@@ -10,8 +10,8 @@ There are output pin selectors, which select which block output is connected to
 The selector is one-hot, so you need to write `8'b100` if you want to select input 3 for example.
 The default value for all of these selectors is `'b10`.
 
-| Address | Pin output | Possible block inputs |
-|---------|------------|-----------------------|
+| Address | Pin output | Possible block outputs |
+|---------|------------|------------------------|
```

Comment:
This also needs to go in the template right? Please use top gen py to render this file.

Reply:
Oops. OK, I'll have a go at figuring-out topgen